dir.props

```diff
@@ -21,6 +21,13 @@
     <RoslynPackageName>Microsoft.Net.ToolsetCompilers</RoslynPackageName>
   </PropertyGroup>
 
+  <!-- Profile-based optimization data package versions -->
+  <PropertyGroup>
+    <!-- TODO: Fill in with actual versions from the right branches -->
```

Comment:
FYI: We are generating counts packages today and I plan to resolve this TODO before merging. If that gets missed, nothing breaks, and it's a trivial follow-up PR to update this.